libraries/llvm/rts.ll

```diff
@@ -403,7 +436,7 @@ define private %Stack @newStack(%Prompt %prompt) {
     ; TODO find actual size of stack
     %stack = call ptr @malloc(i64 120)
 
-    ; TODO initialize to zero and grow later
+    ; TODO initialize to zero?
```

Comment:
Do we actually want to initialize the size to zero?

Reply:
Please no! Ideally, try a few sizes and perform benchmark.